internal/driver/mobile/app/android.go

```diff
@@ -268,6 +269,7 @@ func onConfigurationChanged(activity *C.ANativeActivity) {
 
 //export onLowMemory
 func onLowMemory(activity *C.ANativeActivity) {
+	debug.FreeOSMemory()
```

Comment:
I just recall that we could also call runtime.GC() before manually free OS memories so this is more effective:

```suggestion
	runtime.GC()
	debug.FreeOSMemory()
```